executor/bind.go

```diff
@@ -69,7 +71,19 @@ func (e *SQLBindExec) dropSQLBind() error {
 	return domain.GetDomain(e.ctx).BindHandle().DropBindRecord(record)
 }
 
-func (e *SQLBindExec) createSQLBind() error {
+func (e *SQLBindExec) createSQLBind(ctx context.Context) error {
+	sqlExec := e.ctx.(sqlexec.SQLExecutor)
+	// Use explain to check the validity of bind sql.
+	recordSets, err := sqlExec.Execute(ctx, fmt.Sprintf("explain %s", e.bindSQL))
```

Comment:
`explain` is much more expensive than parsing and plan building?

Reply:
Yes, but it much more easier by using `explain`, and it could also check the privilege of select tables.